src/serialization/CCLACSets.jl

```diff
@@ -0,0 +1,180 @@
+module CCLACSets
+export read_ccl_acset
+
+using ...ACSetInterface, ...Schemas, ..ACSetSerialization, ...DenseACSets
+
+using ACSets
+
+using JSON3
+using MLStyle
+
+@data AnElement begin
+    ObjectElem(name::String, type::String)
+    HomElem(name::String, dom::String, cod::String)
+end
+
+struct CCLModel
+    name::Symbol
+    elems::Dict{String, AnElement}
```

Comment:
Would suggest using the `UUID` type from `Base` for IDs and `String` for human-readable names. This has the advantage that the types alone will make it obvious what kind of "name" you're dealing with.